api/v1alpha1/sentinel_types.go

```diff
@@ -188,18 +190,69 @@ type MonitoredShard struct {
 	Name string `json:"name"`
 	// Master is the address of the master redis server of
 	// this shard, in the format "127.0.0.1:6379"
+	// DEPRECATED - this field will be removed in an upcoming release
 	// +operator-sdk:csv:customresourcedefinitions:type=status
-	Master string `json:"master"`
+	// +optional
+	Master string `json:"master,omitempty"`
```

Comment:
I made it optional to be able to delete it in a later release when it is empty, as right now is a mandatory field in use in the clusters.